homeassistant/components/device_tracker/__init__.py

```diff
@@ -5,37 +5,37 @@
 https://home-assistant.io/components/device_tracker/
 """
 import asyncio
-from datetime import timedelta
 import logging
 import os
 import threading
-from typing import Any, Sequence, Callable
+from datetime import timedelta
+from typing import Any, Callable, Sequence
 
 import voluptuous as vol
 
-from homeassistant.bootstrap import (
-    prepare_setup_platform, log_exception)
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util as util
+import homeassistant.util.dt as dt_util
+from homeassistant.bootstrap import log_exception, prepare_setup_platform
 from homeassistant.components import group, zone
 from homeassistant.components.discovery import SERVICE_NETGEAR
 from homeassistant.config import load_yaml_config_file
+from homeassistant.const import (
+    ATTR_ENTITY_ID, ATTR_GPS_ACCURACY, ATTR_LATITUDE, ATTR_LONGITUDE,
+    DEVICE_DEFAULT_NAME, STATE_HOME, STATE_NOT_HOME)
 from homeassistant.exceptions import HomeAssistantError
 from homeassistant.helpers import config_per_platform, discovery
 from homeassistant.helpers.entity import Entity
-from homeassistant.helpers.typing import GPSType, ConfigType, HomeAssistantType
-import homeassistant.helpers.config_validation as cv
-import homeassistant.util as util
+from homeassistant.helpers.event import track_utc_time_change
+from homeassistant.helpers.typing import ConfigType, GPSType, HomeAssistantType
 from homeassistant.util.async import run_coroutine_threadsafe
-import homeassistant.util.dt as dt_util
 from homeassistant.util.yaml import dump
 
-from homeassistant.helpers.event import track_utc_time_change
-from homeassistant.const import (
-    ATTR_GPS_ACCURACY, ATTR_LATITUDE, ATTR_LONGITUDE,
-    DEVICE_DEFAULT_NAME, STATE_HOME, STATE_NOT_HOME, ATTR_ENTITY_ID)
-
 DOMAIN = 'device_tracker'
 DEPENDENCIES = ['zone']
 
+REQUIREMENTS = ['netaddr==0.7.18']
```

Comment:
it is :) 

https://github.com/home-assistant/home-assistant/pull/4401/files#diff-d2a98e4969c1e91ad53f86f06b98fba2R281